pytorch_lightning/trainer/data_loading.py

```diff
@@ -467,18 +467,8 @@ def request_dataloader(self, model: 'pl.LightningModule', stage: str) -> DataLoa
             The dataloader
         """
         self.call_hook(f"on_{stage}_dataloader")
-        dataloader: DataLoader = getattr(model, f'{stage}_dataloader')()
-        dataloader = self._flatten_dl_only(dataloader)
+        dataloader = getattr(model, f'{stage}_dataloader')()
+        if isinstance(dataloader, tuple):
+            dataloader = list(dataloader)
         self.accelerator.barrier('get_dataloaders')
         return dataloader
-
-    def _flatten_dl_only(self, dataloaders):
-        # handles user error when they return:
-        # return dl1, dl2  vs  return (dl1, dl2)
```

Comment:
don't you think it was meant to handle something like `(dl1, dl2), `
not the extra comma
a function named flatten_dl would make sense in this case, even though the implementation here doesn't handle it like htat.

Reply:
Looking at the PR that introduced it